R/net.mod.nwupdate.R

```diff
@@ -75,7 +75,7 @@ nwupdate.net <- function(dat, at) {
     }
 
     ## Recovery
-    idsRecov <- dat$nw.update$rec$idsRecov
+    idsRecov <- dat$temp$idsRecov
     recovState <- ifelse(dat$control$type == "SIR", "r", "s")
```

Comment:
Can we figure out a way to do this without having to write out idsRecov? One solution, is to look up, in the nwupdate fx the setdiff of ids with a recov status on attr versus the network object.

Reply:
I was going to suggest a time recovered attr like for infection (time infected), but the above works too without needing to add another tracker.